cmd/nvidia-device-plugin/main.go

```diff
@@ -126,6 +126,12 @@ func main() {
 			Usage:   "the path on the host where MPS-specific mounts and files are created by the MPS control daemon manager",
 			EnvVars: []string{"MPS_ROOT"},
 		},
+		&cli.StringFlag{
+			Name:    "mode",
```

Comment:
Remind me what `mode` implies again. Is it the "device discovery mode"? Is there a reason to not use the longer name for clarity?

Reply:
Yes, this is how devices are discovered. I think using a longer name is better. Will update to `device-discovery-mode` instead.

Reply:
Does using `device-discovery-strategy` makes sense here instead? This would better align with the other similar flags.